mowgli_etl/pipeline/wdc/parsimonious_parser/offers_corpus_english_v2_1000_parsed.csv

```diff
@@ -1,53 +1,50 @@
-source
```

Comment:
Looks better.

We'll want to copy this out, probably to Google Drive, and then add annotations. Think about the annotations that make sense. We can have partially correct parses, for example.

We definitely need to isolate the text(s) that's being parsed, the raw_text column is too big. (Retain that but put it at the end.)